packages/cli/code-pushup.config.ts

```diff
@@ -0,0 +1,87 @@
+import 'dotenv/config';
+import { join } from 'path';
+import { z } from 'zod';
+import type { CoreConfig } from '@code-pushup/models';
```

Comment:
You realize this doesn't give any type-safety, right? 

![image](https://github.com/code-pushup/cli/assets/34691111/0a28967e-2fa8-474a-88ba-0ab21a814696)

The package obviously isn't (and shouldn't be) installed, and for the TS path aliases to work the `code-pushup.config.ts` would have to part of some `tsconfig` (would require a special `tsconfig.code-pushup.json`, which is probably overkill). Simple solution is to import type from source directly:

```suggestion
import type { CoreConfig } from '../models/src';
```